src/routes.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+Route::group(['prefix' => 'api'], function () {
+    Route::group(['prefix' => 'pages'], function () {
+    });
+});
```

Comment:
If you don't use intermediate endpoints between api group and pages then you can put it in one group
```suggestion
Route::group(['prefix' => 'api/pages'], function () {
});
```

Reply:
Just wanted to be consistent with the other code we have.